src/charm.py

```diff
@@ -35,6 +36,17 @@
 ETCD_CA_PATH = os.path.join(CALICO_CTL_PATH, "etcd-ca")
 
 
+def conctl_stop(container):
+    """Stop a container with runc independent conctl."""
+    ctl = getContainerRuntimeCtl()
+    try:
+        proc = ctl.delete(container)
+    except CalledProcessError as e:
+        if f"{container} not found" not in e.stderr.decode():
+            raise
```

Comment:
What if we're upgrading to version 1.28 or higher and the `calico-node` container is missing? In such a case, I believe we should proceed ahead even if we can't kill the container.

Reply:
how's the new adjustment. 

if the container isn't found -- then it's definitely stopped  and we return `True`

Reply:
Looks great, thanks!